apps/storybook/.storybook/pclnTheme.js

```diff
@@ -1,10 +1,17 @@
-import { create } from '@storybook/theming/create'
+import { create } from '@storybook/theming'
+import { createTheme } from 'pcln-design-system'
+
+const theme = createTheme()
+
+// https://storybook.js.org/docs/react/configure/theming#create-a-theme-quickstart
 
 export default create({
-  base: 'light',
+  base: 'dark',
```

Comment:
I think it should default to light since that's the mode that Priceline is in by default